docs/Inventory of Epics.md

```diff
@@ -0,0 +1,25 @@
+# "Inventory of Epics" for the Roslyn Analyzer Ecosystem
+
+Here is a current list of the major areas of work ("epics") for the Roslyn analyzer ecosystem. See the [README](https://github.com/dotnet/roslyn-analyzers-contrib/blob/master/README.md) for an explanation of what that means.
+
+1. Replace FxCop
+
+    Replace FxCop with a collection of Roslyn-based analyzers and fixers. We will take the opportunity to remove FxCop rules that are no longer relevant, and to add new rules around new language features.
+
+2. Provide style analyers in VS
+
+    Create a comprehensive, configurable set of Roslyn-based style analyzers and fixers. These will draw on the work of @sharwell.
+
+3. Implement a configuration mechanism for Roslyn analyzers and fixers
+
+    This is especially important for style analyzers, because different projects have different "house styles".
+
+4. Improve the test infrastructure
```

Comment:
:thought_balloon: I think this point should focus on improving the templates used to create new analyzer projects. Test infrastructure improvements are only one aspect of this (though an important one).